storage/rocksdb/ha_rocksdb.cc

```diff
@@ -292,6 +292,42 @@ static int rocksdb_force_flush_memtable_now(
   return HA_EXIT_SUCCESS;
 }
 
+static void rocksdb_force_flush_memtable_and_lzero_now_stub(
+    THD *const thd, struct st_mysql_sys_var *const var, void *const var_ptr,
+    const void *const save) {}
+
+static int rocksdb_force_flush_memtable_and_lzero_now(
+    THD *const thd, struct st_mysql_sys_var *const var, void *const var_ptr,
+    struct st_mysql_value *const value) {
+  sql_print_information("RocksDB: Manual memtable and L0 flush\n");
+  rocksdb_flush_all_memtables();
+
+  const Rdb_cf_manager &cf_manager = rdb_get_cf_manager();
+  for (const auto &cf_handle : cf_manager.get_all_cf()) {
+    rocksdb::ColumnFamilyMetaData metadata;
+    rdb->GetColumnFamilyMetaData(cf_handle, &metadata);
+
+    std::vector<std::string> file_names;
+    for (auto &file : metadata.levels[0].files) {
+      DBUG_ASSERT(metadata.levels[0].level == 0);
+      file_names.emplace_back(file.db_path + file.name);
```

Comment:
Originally I did put a `/` there.  @hermanlee commented above asking if it was necessary, I stepped through GDB and found that it was not.  See my comment above for more details.